domains/authorization/Authorization.API/API/ValueObjects/OrganizationName.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Text.RegularExpressions;
+
+namespace API.ValueObjects;
+
+public readonly partial record struct OrganizationName
```

Comment:
Make ValueObject base object instead of record and inherrit from that

Reply:
Do we have any base configuration we can throw in the Base Value object?

Reply:
I don't understand the question

Reply:
I added a Base ValueObject